src/mcts/params.cc

```diff
@@ -164,6 +164,14 @@ const OptionId SearchParams::kHistoryFillId{
     "one. During the first moves of the game such historical positions don't "
     "exist, but they can be synthesized. This parameter defines when to "
     "synthesize them (always, never, or only at non-standard fen position)."};
+const OptionId SearchParams::kCertaintyPropagationId{
+    "certainty-propagation", "CertaintyPropagation", 
+    "Propagates certain scores more efficiently in the search tree, "
+    "proves and displays mates."};
+const OptionId SearchParams::kTwoFoldDrawScoringId{
+    "two-fold-draw-scoring", "TwoFoldDrawScoring",
```

Comment:
`twofold-is-draw` ?

Reply:
It is "scored" as a draw in search, its not terminating the game.